lambdas/sharedattributes/src/main/java/uk/gov/di/ipv/core/sharedattributes/SharedAttributesHandler.java

```diff
@@ -77,17 +77,17 @@ private SharedAttributesResponse getSharedAttributes(String ipvSessionId)
         Map<String, String> credentials =
                 userIdentityService.getUserIssuedCredentials(ipvSessionId);
 
-        List<SharedAttributes> sharedAttributes = new ArrayList<>();
+        List<SharedAttributes> sharedAttributesNew = new ArrayList<>();
```

Comment:
Can we drop the use of `New`? As it loses context once merged and is hard to understand.